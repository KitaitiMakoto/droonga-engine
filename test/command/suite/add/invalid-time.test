#@include fixture/user-table.jsons
{
  "type": "add",
  "dataset": "Droonga",
  "body": {
    "table": "User",
    "key": "key",
    "values": {
      "age": "secret"
    }
  }
}
