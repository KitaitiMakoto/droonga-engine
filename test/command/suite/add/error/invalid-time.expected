[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 400,
    "type": "add.result",
    "body": {
      "name": "InvalidValue",
      "message": "The column \"birthday\" cannot store the value \"today\".",
      "detail": {
        "table": "User",
        "key": "key",
        "values": {
          "birthday": "today"
        }
      }
    },
    "errors": {
      "sources0": {
        "status_code": 400,
        "body": {
          "name": "InvalidValue",
          "message": "The column \"birthday\" cannot store the value \"today\".",
          "detail": {
            "table": "User",
            "key": "key",
            "values": {
              "birthday": "today"
            }
          }
        }
      },
      "sources1": {
        "status_code": 400,
        "body": {
          "name": "InvalidValue",
          "message": "The column \"birthday\" cannot store the value \"today\".",
          "detail": {
            "table": "User",
            "key": "key",
            "values": {
              "birthday": "today"
            }
          }
        }
      }
    }
  }
]
